slides/02-data-budget.qmd

```diff
@@ -282,27 +287,23 @@ ggplot(frog_train, aes(age, latency, color = reflex)) +
 
 ```{r echo = FALSE}
 #| fig-align: 'center'
-quartiles <- quantile(frog_train$latency, probs = c(1:3)/4)
-ggplot(frog_train, aes(latency)) +
-  geom_histogram(bins = 20) +
-  geom_vline(xintercept = quartiles, color = train_color, 
-             size = 1.5, lty = 2)
+table(taxi_train$tip)
```

Comment:
if we are going away from a plot, I would suggest we use `count()` 

```r
#| echo: true
taxi_train %>%
  count(tip)
```

Reply:
`count()` -> `table()` in [c9fb844](https://github.com/tidymodels/workshops/pull/108/commits/c9fb8449b82314ba875078b7d5b0351a3a141b5e) (before I saw Hannah's comment!🙈)

I'm game for a plot. Will think about how to visualize this.

Reply:
[deba984](https://github.com/tidymodels/workshops/pull/108/commits/deba984a1b2ea4747d1e943cef3b992d56502817) :)